test/clustertest/tests/BadCommandTest.cpp

```diff
@@ -10,7 +10,7 @@ struct BadCommandTest : tpunit::TestFixture {
     BedrockClusterTester* tester;
 
     void setup() {
-        tester = new BedrockClusterTester(_threadID, "");
+        tester = new BedrockClusterTester("" /*Explicitly exclude the default plugins. TODO: Why does this help?*/);
```

Comment:
hmm, what? 😅

Reply:
also _what_ does it help? lol

Reply:
The test passes with `""` and fails without it.

Why? I don't know. I removed it cause it seemed unnecessary, then put it back with the comment because there were too many other things to work on. If only I'd added a better comment when I wrote it in the first place.

Reply:
Yeah, can you update this comment to say that?
```
Explicitly exclude the default plugins. This allows tests to pass and fails without it, and we don't know why.
```